### DISCUSSION TITLE
Plugins I've made

### DISCUSSION THREAD

**Discussion by kran27** [rejetto/hfs#507] [Show and tell]
I'm sure there are numerous code improvements that could be made to all of these. I'm pretty new to both HFS and JavaScript/web stuff in general.

First, [my theme](https://github.com/kran27/e-z-theme)
![image](https://github.com/rejetto/hfs/assets/56488228/bd6c1605-8e3c-4e30-abb0-3a7f5ddaa2be)
![image](https://github.com/rejetto/hfs/assets/56488228/ec5c7100-6f7c-4ebe-aa03-f77fb75e3571)
[Code viewer with highlighting](https://github.com/kran27/code-preview)
![image](https://github.com/rejetto/hfs/assets/56488228/4369d2a7-2c24-42f4-b2b3-c7a3f04a2fec)
and finally (for now), [Webamp](https://github.com/kran27/webamp-plugin)
![image](https://github.com/rejetto/hfs/assets/56488228/a00e74a3-dd0b-42a2-bc58-8adadcab488a)
includes theme support!
![image](https://github.com/rejetto/hfs/assets/56488228/f2c60275-f084-4645-aa4e-693f8242e6e3)

**Comment by rejetto** [comment]
kudos :)

**Comment by PH5HP** [comment]
Hi,

I installed and activate the Webamp plugin, but how does it work? I am still getting the normal webplayer instead of WebAmp. Could you please give some more indormation about how it works? 

The same for the codeviewer plugin. It seems that every still has been opened in some kind of default viewer of the browser.

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Reply by kran27** [reply]
Both plugins get supported types from a regex string in the config. Code files get a "Show" option in the file menu, audio files get a "Load Into Webamp" option. Also make sure you're on Webamp 1.01.

**Reply by PH5HP** [reply]
Hi,

Exactly what I wanted to know. Thank you.

Kind regards,
https://fileserver.ph5hp.nl (when my computer is online)
